App.jsx

```diff
@@ -0,0 +1,10 @@
+const App = () => {
+	return (
+		<div>
+			<h1 className='header'>User Cards</h1>
```

Comment:
No big deal, but for some reason the whitespace is showing up kind of weird on Github. It looks fine in my text editor - not a big deal or anything, just something to be mindful of as you work on portfolio projects!